pkg/cedar/core/system_summary.go

```diff
@@ -78,6 +86,7 @@ func (c *Client) GetSystemSummary(ctx context.Context, opts ...systemSummaryPara
 				SystemMaintainerOrg:     zero.StringFrom(sys.SystemMaintainerOrg),
 				SystemMaintainerOrgComp: zero.StringFrom(sys.SystemMaintainerOrgComp),
 				ID:                      zero.StringFromPtr(sys.IctObjectID),
+				UUID:                    zero.StringFrom(uuid.String()),
```

Comment:
Should we use `nil` if this fails to parse? Otherwise the FE will get [a uuid of zeroes](https://goplay.tools/snippet/hgQsU-L8foJ).

Reply:
Oh good catch, idk why I thought UUID.parse returned empty string. Lemme address that

Reply:
https://github.com/CMSgov/easi-app/pull/2599/commits/f057afff01f7eb1be0854450b0d8746540dd032c